src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -22,6 +22,11 @@ public class ApplicationInsightsLoggerOptions : IOptionsFormatter
         /// </summary>
         public SamplingPercentageEstimatorSettings SamplingSettings { get; set; }
 
+        /// <summary>
+        /// Gets or sets excluded types for sampling.
+        /// </summary>
+        public string SamplingExcludedTypes { get; set; }
```

Comment:
While we're in here -- I think we should just add `IncludedTypes` as well to give people the full API.